src/Symfony/Component/Ldap/LdapFactory.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Ldap;
+
+use Symfony\Component\Ldap\Exception\DriverNotFoundException;
+
+/**
+ * @author Charles Sarrazin <charles@sarraz.in>
+ */
+final class LdapFactory
```

Comment:
The class is simply a shorthand way to create the Ldap class. Indeed, otherwise, you need to manually inject the adapter (and other configuration). This is basically to improve DX.